ddl/ddl_api.go

```diff
@@ -4942,6 +4952,17 @@ func (d *ddl) AlterTableSetTiFlashMode(ctx sessionctx.Context, ident ast.Ident,
 		return fmt.Errorf("unsupported TiFlash mode %s", mode)
 	}
 
+	err = checkTiFlashSupportable(schema, tb)
+	if err != nil {
+		return errors.Trace(err)
+	}
+
+	// Prompt warning when tiflash replica is nil or tiflash replica count is 0
```

Comment:
This comment does not provide any useful information, it describes what code is already describing. Maybe you will want:

```
// Prompt warning when there is no TiFlash replica, as TiFlash mode will only take effect when executing in TiFlash.
```